project/devs.sls

```diff
@@ -3,7 +3,7 @@ include:
 
 {% if 'users' in pillar and pillar['users'] %}
 {% for user, args in pillar['users'].iteritems() %}
-{{ user }}:
+dev_{{ user }}:
```

Comment:
Yes I would think so.  Strange that it didn't break when I tested it. I'll make that change and see what happens.